rust/Cargo.toml

```diff
@@ -6,4 +6,7 @@ members = [
 ]
 
 [profile.release]
-debug = false
\ No newline at end of file
+debug = false
+
+[patch.crates-io]
+bellman = { git = "https://github.com/dannasessha/bellman", rev="712337ea591cb89e031b9f203462d32de00bf7ff" }
```

Comment:
Shall we fork `bellman` into zingolabs?

Reply:
Sure, no reason why not if we're going to possibly use this patch.